src/slack_connector.ts

```diff
@@ -231,6 +231,22 @@ export class SlackConnector implements IConnector {
     return client.im.open(slackUserId)
   }
 
+  public async startThread(message: IMessage): Promise<IAddress> {
```

Comment:
I think it would be a better idea to use the `send` method inside this implementation. Probably the send should do this by default ?